### PR TITLE
[PLATFORM-1243]: Fix interaction with multiclient auth0_ex

### PR DESCRIPTION
**Pull request by cpiemontese**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-1243

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,12 @@ Then configure your bridge with the audience of the target service:
 use BridgeEx.Graphql,
   endpoint: "...",
   decode_keys: :strings,
-  auth0: [enabled: true, audience: "target_audience"]
+  auth0: [
+    enabled: true,
+    audience: "target_audience",
+    # You can optionally specify a client (in case you defined multiple ones)
```
Comment (cottinisimone) [inline README.md:154]: What about saying that the default client will be used otherwise?

Comment (cpiemontese) [reply README.md:154]: Wdyt?

File: lib/auth0_authorization_provider.ex
Comment (cpiemontese) [inline lib/auth0_authorization_provider.ex:1]: Don't like this, I accept suggestions

File: README.md (hunk 1 above)
Comment (cottinisimone) [reply README.md:154]: :+1:

**Review by cottinisimone — APPROVED** [review]

**Review by cottinisimone — APPROVED** [review]

**Pull request merged**